files/sysbus/org.webosports.service.audio.api.json

```diff
@@ -271,6 +271,7 @@
     "org.webosports.service.audio/ringtone/setMuted",
     "org.webosports.service.audio/ringtone/setVolume",
     "org.webosports.service.audio/ringtone/status",
+    "org.webosports.service.audio/setCallMode",
```

Comment:
Seems I missed that one somehow. Might be good to add this one as well, while we're at it? https://github.com/webOS-ports/audio-service/blob/webOS-ports/webOS-OSE/src/audioservice.cpp#L66